### ISSUE TITLE
Inconsistent match color used in LSP workspace symbols

### ISSUE THREAD

**Issue by MariaSolOs** [ibhagwan/fzf-lua#1062]
### Info

- Operating System: Fedora Linux Asahi Remix
- Shell: fish
- Terminal: WezTerm
- `nvim --version`: v0.10.0-dev-c2ddef30e
- `fzf --version`: 0.44.1

### Description

Not sure if this is a bug, but it seems that the color used to highlight matches in the LSP workspace symbols picker differs from the one used by other pickers, such as document symbols and find files.

Notice how in the picture below, matches are highlighted in red:
![image](https://github.com/ibhagwan/fzf-lua/assets/62502207/d6301736-025f-4f9a-a301-88d0a1323ce6)

While matches in document symbols use a different color:
![image](https://github.com/ibhagwan/fzf-lua/assets/62502207/c7640521-52c1-4763-bc2a-ac7fced11fd0)

From a brief exploration, I think this is setup here, which is not configurable:
https://github.com/ibhagwan/fzf-lua/blob/785e7db2d5db9d173c8a5718df5c5219c9196352/lua/fzf-lua/providers/lsp.lua#L203

For consistency, I would like all matches to use the same colors to highlight matches :)
I'm aware that colors are further configurable via `fzf` and `ripgrep`, but in this particular case I think (?) fzf-lua entirely handles the colorization.

**Comment by ibhagwan** [comment]
> For consistency, I would like all matches to use the same colors to highlight matches :)
I'm aware that colors are further configurable via fzf and ripgrep, but in this particular case I think (?) fzf-lua entirely handles the colorization.

Related #1028, the reason I made it this way is to make the live workspace symbols consistent with live grep, with `live_grep` the results are colored by `rg` itself with the terminal red color so after #1028 I opted to making live symbols behave the same.

That said, terminal colors aren’t hard coded when it comes to the actual RGB value, these are controlled by your colors scheme with `vim.g.terminal_color_{0-15}`.

**Comment by ibhagwan** [comment]
Also think about running a live grep for `foo` and then pressing ctrl-g to refine using fuzzy, the regex `foo` will be highlighted in red and the fuzzy matching would now be highlighted in different color, the same applies for live workspace symbols into fuzzy matching, what you’re asking would remove the extra info of the live query matches.

**Comment by MariaSolOs** [comment]
> Related #1028, the reason I made it this way is to make the live workspace symbols consistent with live grep, with `live_grep` the results are colored by `rg` itself with the terminal red color so after #1028 I opted to making live symbols behave the same.

Sounds reasonable!

> That said, terminal colors aren’t hard coded when it comes to the actual RGB value, these are controlled by your colors scheme with `vim.g.terminal_color_{0-15}`.

Right, but I don't want to change my terminal red color, I'm just interested in modifying this particular highlight used by fzf-lua.

This is still quite low priority though, so I'm completely fine if you want to close this issue! Promise it won't make me love the plugin any less.

**Comment by ibhagwan** [comment]
> Right, but I don't want to change my terminal red color, I'm just interested in modifying this particular highlight used by fzf-lua.
> This is still quite low priority though, so I'm completely fine if you want to close this issue! Promise it won't make me love the plugin any less.

Now that makes it two requests for the same thing, if I’m being honest I was probably a bit lazy to do it when @I-Own-You asked in https://github.com/ibhagwan/fzf-lua/issues/1028#issuecomment-1927384846, I’ll leave this open until I add a custom hl for this.

**Comment by ibhagwan** [comment]
@MariaSolOs, @I-Own-You, try https://github.com/ibhagwan/fzf-lua/commit/85c09721b3145fdf452e082ca0506943362667d5

You can either configure it in your colorscheme, for example:
```lua
:hi! link FzfLuaLiveSym IncSearch
```

Or temporarily set it for a single run:
```lua
:FzfLua lsp_live_workspace_symbols hls.live_sym=IncSearch
```

Or permatently configure it in setup under:
```lua
require('fzf-lua').setup{
  hls = { live_sym = "IncSearch" }
}
```

Resulting in:
![image](https://github.com/ibhagwan/fzf-lua/assets/59988195/0d5c4605-15d3-4fa7-b01a-1b3b713bbaa1)

**Comment by MariaSolOs** [comment]
@ibhagwan thank you so much! <3
